public_header.go

```diff
@@ -216,8 +217,8 @@ func ParsePublicHeader(b *bytes.Reader, packetSentBy protocol.Perspective) (*Pub
 	return header, nil
 }
 
-// GetLength gets the length of the publicHeader in bytes
-// can only be called for regular packets
+// GetLength gets the length of the publicHeader in bytes.
+// It can only be called for regular packets.
```

Comment:
I don't have an opinion on that, but since we are stating quite clearly that it's unstable I would leave it in for now, since I don't see any advantage in privatizing it.